test/kube_testing/k8s.go

```diff
@@ -144,8 +145,24 @@ func blockUntilPodReady(client kubernetes.Interface, timeout time.Duration, sour
 			break
 		}
 
-		if time.Since(st) > timeout/2 {
+		if time.Since(st) > timeout/2 && !infoPrinted {
 			logrus.Infof("Pod deploy half time passed: %v", pod)
+
+			cmd := exec.Command("kubectl", "get", "pods", "-o", "wide")
```

Comment:
Is there a better way? We deliberately left the `shell `script` world.

Reply:
Removed, I've not found a better way at the moment. Kubernetes API now show all messages some times, not sure why.